modules/dnn/src/layers/bitshift_layer.cpp

```diff
@@ -0,0 +1,138 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+// Copyright (C) 2025, BigVision LLC, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include "../precomp.hpp"
+#include "layers_common.hpp"
+
+// ONNX reference: BitShift operator
+// Spec: https://onnx.ai/onnx/operators/onnx__BitShift.html
+// Supported opsets: ai.onnx opset 11 and newer
```

Comment:
please, put a note here that broadcasting is not fully supported, only same shapes (`array shift array`) or `array shift scalar`